packages/react-ui-ag/src/Listings/MobileMapListing.js

```diff
@@ -142,6 +154,30 @@ export default class MobileMapListing extends Component {
     )
   }
 
+  renderPhotoCarousel() {
+    const { theme, lazyLoad, photos } = this.props
+    const photoProps = {
+      showNav: true,
+      ...photos,
+    }
+
+    if (lazyLoad) {
+      photoProps.lazyLoad =
+        typeof lazyLoad === 'boolean'
+          ? REACT_LAZYLOAD_PROP_DEFAULTS
+          : lazyLoad
+    }
+
+    return (
+      <div className={theme.MobileMapListing_Top}>
+        <ListingComponents.Photos
+          {...photoProps}
+          className={theme.MobileMapListing_Photos}
+        />
+      </div>
+    )
```

Comment:
I think a cleaner approach would be something like this IMO.  Thoughts?
```
const { theme, photos } = this.props
let lazyLoad = this.props.lazyLoad

if (lazyLoad && typeof lazyLoad === 'boolean') {
  lazyLoad = REACT_LAZYLOAD_PROP_DEFAULTS
}

return (
  <div className={theme.MobileMapListing_Top}>
    <ListingComponents.Photos
      showNav
      photos={photos}
      lazyLoad={lazyLoad}
      className={theme.MobileMapListing_Photos}
    />
  </div>
)

```

Also, is there a way to remove the outer (theme.MobileMapListing_Top) div?  Maybe pass the className directly to `ListingComponents.Photos`?

Reply:
Much cleaner. Thanks I'll update this. However, I think we need to refactor ListingComponents or actually how we distribute props in ListingCarousel. It's kinda confusing what's getting destructured because of duplicate prop names etc.